### ISSUE TITLE
quando eu disparo:  http://localhost:8080/api/clientes

### ISSUE THREAD

**Issue by DevJoseWeb** [m4rciosouza/ponto-inteligente-api#2]
Ele pede usuário e senha. Startei o MongoDb., mesmo assim ! essa senha é do TomCat ?

`{
"timestamp": 1516034256945,
"status": 401,
"error": "Unauthorized",
"message": "Bad credentials",
"path": "/api/clientes"
}
`

![image](https://user-images.githubusercontent.com/488409/34952824-2ab64300-fa02-11e7-806b-fb55584a7053.png)

**Comment by m4rciosouza** [comment]
Essa é a autenticação basic do Spring Boot, o usuário é "user" e a senha é exibida dinamicamente no console ao iniciar a aplicação.

**Comment by matheuslima5926** [comment]
Tem como desabilitar essa autenticação padrão?

**Comment by m4rciosouza** [comment]
Sim, adicione ao application.properties a seguinte configurarção:

security.basic.enabled=false
